e2e/e2e_sp_exit_test.go

```diff
@@ -0,0 +1,148 @@
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"github.com/bnb-chain/greenfield-go-sdk/e2e/basesuite"
+	"github.com/bnb-chain/greenfield-go-sdk/types"
+	storageTestUtil "github.com/bnb-chain/greenfield/testutil/storage"
+	spTypes "github.com/bnb-chain/greenfield/x/sp/types"
+	storageTypes "github.com/bnb-chain/greenfield/x/storage/types"
+	"github.com/stretchr/testify/suite"
+
+	types3 "github.com/bnb-chain/greenfield/x/sp/types"
+)
+
+type BucketMigrateTestSuite struct {
```

Comment:
Shall we rename the test file name from `e2e/e2e_sp_exit_test.go` to `e2e/e2e_migrate_bucket_test.go`